test/benchmark/benchmark.go

```diff
@@ -153,6 +188,7 @@ func readCurrentBenchmarkResults() benchmarkResult {
 }
 
 func tearDownWeavaite() error {
```

Comment:
there's a misspelling should be `tearDownWeaviate`

Reply:
thx, fixed